src/common/mfu_io.c

```diff
@@ -251,8 +506,33 @@ int mfu_hardlink(const char* oldpath, const char* newpath)
 }
 
 /*****************************
- * Files
- ****************************/
+ *  * Files
```

Comment:
There are a few spots like this where the editor is dropping in some extra * characters

Reply:
@adammoody ok, will fix those.